src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -363,14 +409,14 @@ void compare(const ov::Tensor& expected,
             throw std::runtime_error(out_stream.str());
         }
         double abs = std::fabs(expected_value - actual_value);
-        double rel = expected_value ? (abs / std::fabs(expected_value)) : abs;
+        double rel = expected_value && !std::isinf(expected_value) ? (abs / std::fabs(expected_value)) : abs;
```

Comment:
In my understanding: we can not use relative error if value is zero. 

Imagine next situations:
case #1: expected = 0.0, actual = 0.1, rel = 0.1 <= only 10%, it's not correct
case #2: expected = 0.05, actual = 0.1, rel = abs(0.05 - 0.1)/0.05= 19 <= 1900%

In the case #2 difference is smaller, but `rel` is much larger then in case #1. It's not correct. Don't use relative error if value is zero.

Reply:
Changed, rel error is calculated now as:
```
double rel =
            expected_value && actual_value && !std::isinf(expected_value) ? (abs / std::fabs(expected_value)) : 0;
```